config/settings.yml

```diff
@@ -116,3 +116,5 @@ rails_performance:
 
 file_storage:
   default_root_path: <%= ::Rails.root %>/data/<%= ::Rails.env %>/
+
+rmd_convert_enabled: false
```

Comment:
This is good; you also should add this option to the *schema* defined in `config.rb`